src/Draco.LanguageServer/DracoDocumentRepository.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using static Draco.Compiler.Api.Syntax.ParseTree.Expr;
+
+namespace Draco.LanguageServer;
+
+internal static class DracoDocumentRepository
```

Comment:
Don't make a service static, use DI to inject the same instance for both handlers